using-timescaledb/continuous-aggregates.md

```diff
@@ -96,6 +96,120 @@ with PostgreSQL. This means that aggregates with `ORDER BY` clauses or `DISTINCT
 additionally we require that no `FILTER` clauses appear ([though we will allow them in future
 releases](#future-work)).
 
+### Integer-based Continuous Aggregates [](create-integer)
+
+Usually, continuous aggregates are defined on a time column, but it is
```

Comment:
Great example! The only suggestion is to move this under a new section of the tutorial like Advanced Topics/Advanced Users (or something similar).
This is fairly self-contained and is of interest only to certain users. Also, will not affect the current flow of the tutorial.  Maybe Best Practices and this section go under Advanced Topics?

I am not able to add a comment to unchanged parts of the doc. I see a section called Data retention for continuous aggregates. The paragraph about down sampling and ignore_invalidation_older_than (around line 281) could probably be moved to the Data retention section. I think it is a better fit there.

Reply:
> I am not able to add a comment to unchanged parts of the doc. I see a section called Data retention for continuous aggregates. The paragraph about down sampling and ignore_invalidation_older_than (around line 281) could probably be moved to the Data retention section. I think it is a better fit there.

The PR for the additions to the data retention section is #314.

Reply:
> Great example! The only suggestion is to move this under a new section of the tutorial like Advanced Topics/Advanced Users (or something similar).
> This is fairly self-contained and is of interest only to certain users. Also, will not affect the current flow of the tutorial. Maybe Best Practices and this section go under Advanced Topics?

Good idea. I'll add an "Advanced Topics" section. Note that this is not the tutorial, this is the "using" section of the documentation.

Reply:
Moved section to new "Advanced Topics" section.